packages/entity-detail/src/modules/entityDetail/sagas.js

```diff
@@ -188,7 +189,9 @@ export function* loadData() {
   const {entityName, entityId, formName, formDefinition} = yield select(entityDetailSelector)
   const entity = yield call(loadEntity, entityName, entityId, formDefinition, formName)
   const formValues = yield call(form.entityToFormValues, entity)
-  yield put(formActions.initialize(FORM_ID, formValues, {keepDirty: true}))
+  const model = yield call(fetchModel, entityName)
```

Comment:
wouldn't `entityToFormValues` be a good place enhance rather than a new separate function?

Reply:
Functions should have rather one than multiple purposes. So no.